app/assets/javascripts/angular/directives/home.js.erb

```diff
@@ -77,4 +77,31 @@ app.directive("scenarySlider", function() {
         });
       }
     }
+});
+
+app.directive("distanceSlider", function() {
```

Comment:
You should
1. "Refactor" your 4 sliders functions into 1, and use variables to customize each slider when you call them: http://stackoverflow.com/questions/17009276/passing-variable-to-directives-attribute. Maybe you can do this later when you have time, but it's recommended.
2. You should at least give a better name to the file